frontend/src/components/elements/ArrowTable/TheArrowTable.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Table } from "apache-arrow"
+
+type CellType = "blank" | "columnHeading" | "rowHeading" | "data"
+
+interface Cell {
+  type: CellType
+  content: string
+  id?: string
+  classNames: string
+}
+
+class TheArrowTable {
```

Comment:
Can you explain the name of this class? I don't understand the `The` prefix.

Reply:
It was temporary. I'm thinking something like `lib/arrowProto.ts`. Agree?